main.tex

```diff
@@ -1147,8 +1147,70 @@ \section{Beyond linearity}
     f(x+h)\approx f(x) + \nabla f(x) \cdot h + \frac{1}{2} h^\top (Hf)(x) h,
 \end{equation*} 
 where we wrote $d^2 f = Hf$ as the Hessian of $f$ at $x$.
+
+Let us list some useful properties:
+\begin{itemize}
+    \item Set $F$ twice differentiable and define $F_h(u)=dF(u)[h]$. Then,
+    \[ dF_h(u)[k] = F''(u)[h,k] \] 
+    We can then compute the second derivative by fixing $h$ in the first derivative and differentiating like we've done in respect to $u$ for a direction $k$.
+    \item If $F$ is twice differentiable, then $F''(u)\in\mathcal{L}(X\times X, Y)$ is symmetric.
+    \item Partial derivatives are defined using projections, i.e. if $F: U\times V \to Y$, set $\sigma_v(u)=(u,v)$. Then, the partial derivatives w.r.t. $u$ is
+    \[ d(F\circ \sigma_v)(u)[h], \]
+    denoted simply $d_u F(u^*,v^*)$. Notably, one obtains
+    \[ d_u F(u^*,v^*)[h] = d_G(F\circ \sigma_v)(u)[h] = \left.\frac{d}{d\varepsilon}\right|_{\varepsilon = 0} F(u+\varepsilon h, v) \]
+\end{itemize}
+
+\example{
+    Let $\psi(u)=\frac{1}{p}\int_\Omega u^p dx$, defined in $L^p$.
+
+    \begin{align*}
+        d\psi(u)[v] &= \frac{1}{p}\frac{d}{d\varepsilon} |_{\varepsilon=0} \int_\Omega (u+\varepsilon v)^p dx \\
+        &= \frac{1}{p} \int_\Omega p(u+\varepsilon v)^{p-1}v dx|_{\varepsilon=0} \\
+        &= \int_\Omega u^{p-1}v dx
+    \end{align*}
+
+    \begin{align*}
+        d^2\psi(u)[v_1,v_2] &= d(d F_{v_1}(u))[v_2] \\
+        &= \frac{d}{d\varepsilon}|_{\varepsilon=0}\int_\Omega(u+\varepsilon v_2)^{p-1} v_1 dx\\
+        &= (p-1)\int_\Omega u^{p-2}v_1 v_2 dx
+    \end{align*}
+}
+
+In calculus of variations $d^2 F(u)[h] = d^2 F(u)[h,h]$ is used for checking properties of a problem.
+
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Local analysis}
+Consider the set 
+\[ \text{Inv}(X,Y) = \left\{ A\in \mathcal{L}(X,Y): A \text{ is invertible} \right\} \]
+
+Then, we have the following local inversion
```

Comment:
the following local inversion result. Falta "result" (o alguna otra palabra, da la sensación que quedó cortada la frase).